Misc/tools/locust_load_test/locustfile-ui.py

```diff
@@ -0,0 +1,45 @@
+from locust import HttpLocust, TaskSet, task
+
+
+class UserBehavior(TaskSet):
+    @task
+    def getPatients(self):
+        self.client.get("/api/v1/patients")
```

Comment:
~~Maybe api prefix could be extracted to a variable~~ Actually i kinda like it like it is